### PR TITLE
[Improve]Trivial improvements

### PR DESCRIPTION
**Pull request by smallyao**
## What changes were proposed in this pull request

Issue Number: close #xxx 

## Brief change log

Trivial improvements

## Verifying this change

This change is a trivial rework / code cleanup without any test coverage.

This change added tests and can be verified as follows:

## Does this pull request potentially affect one of the following parts
 - Dependencies (does it add or upgrade a dependency):  no

### CODE REVIEW

**Review by caicancai — DISMISSED** [review]
LGTM

File: streampark-common/src/main/scala/org/apache/streampark/common/util/Utils.scala [hunk 1]
```diff
@@ -71,7 +71,14 @@ object Utils extends Logger {
 
   def getJarManifest(jarFile: File): jar.Manifest = {
```
Comment (wolfboys) [inline streampark-common/src/main/scala/org/apache/streampark/common/util/Utils.scala:74]: We can make the code more cleaner:

```
  import org.apache.streampark.common.util.ImplicitsUtils._

  def getJarManifest(jarFile: File): jar.Manifest = {
    requireCheckJarFile(jarFile.toURL)
    new JarInputStream(new FileInputStream(jarFile)).autoClose(_.getManifest)
  }
```

Comment (smallyao) [reply streampark-common/src/main/scala/org/apache/streampark/common/util/Utils.scala:74]: Thanks for your review.
Fixed.

**Comment by smallyao** [comment]
> LGTM

Thanks @caicancai

**Review by wolfboys — APPROVED** [review]

**Pull request merged**